File: testdata/bench/kotlin/BenchmarkTest00013.kt
package bench

import io.ktor.server.application.*
import io.ktor.server.response.*

// Safe: hardcoded query with no user input
suspend fun handler00013(call: ApplicationCall) {
    val conn = java.sql.DriverManager.getConnection("jdbc:sqlite::memory:")
    val rs = conn.createStatement().executeQuery("SELECT COUNT(*) FROM users")
    call.respondText(rs.toString())
}

File: testdata/bench/go/BenchmarkTest00013.go
package bench

import (
	"database/sql"
	"net/http"
)

// Parameterized QueryRow
func Handler00013(w http.ResponseWriter, r *http.Request) {
	param := r.URL.Query().Get("email")
	db, _ := sql.Open("sqlite3", ":memory:")
	db.QueryRow("SELECT id FROM users WHERE email = ?", param)
}
